packages/db/prisma/schema.prisma

```diff
@@ -65,3 +65,12 @@ model GroupInvite {
 
   @@unique([groupId, email])
 }
+
+model EmailVerificationCode {
+  email          String   @id @unique
```

Comment:
I think I missed this before, we actually don't need the @unique here because the @id handles this

You can delete @unique here and remove the part in the migration file which follows the "create index" and it should be good to go